shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/core/database/visitor/SQLVisitorRule.java

```diff
@@ -282,6 +282,11 @@
     
     REPAIR_TABLE("RepairTable", SQLStatementType.DAL),
     
+
```

Comment:
@flycash Please remove this useless blank line.

Reply:
done